modules/hybrid-app/ios/ReactNativeHybridApp.mm

```diff
@@ -13,13 +13,16 @@ - (void)closeReactNativeApp:(BOOL)shouldSignOut shouldSetNVP:(BOOL)shouldSetNVP
 
 - (void)completeOnboarding:(BOOL)status {
     NSLog(@"[ReactNativeHybridApp] `completeOnboarding` should never be called in standalone `New Expensify` app");
-
 }
 
 - (void)switchAccount:(NSString *)newDotCurrentAccountEmail authToken:(NSString *)authToken policyID:(NSString *)policyID accountID:(NSString *)accountID {
     NSLog(@"[ReactNativeHybridApp] `switchAccount` should never be called in standalone `New Expensify` app");
 }
 
+- (void)sendAuthToken:(NSString *)authToken {
+    NSLog(@"[ReactNativeHybridApp] `sendAuthToken` should never be called in standalone `New Expensify` app");
```

Comment:
What about creating a template for those logs, method name could be a parameter🤔

Reply:
Same for Android and web

Reply:
Maybe we could even create single template on JS side and print those logs based on `isHybridApp` value